pkgs/applications/graphics/drawio/default.nix

```
@@ -1,104 +1,90 @@
-{ stdenv, lib, fetchurl, rpmextract, autoPatchelfHook, wrapGAppsHook
-
-# Dynamic libraries
-, alsa-lib, atk, at-spi2-atk, at-spi2-core, cairo, dbus, cups, expat
-, gdk-pixbuf, glib, gtk3, libX11, libXScrnSaver, libXcomposite, libXcursor
-, libXdamage, libXext, libXfixes, libXi, libXrandr, libXrender, libXtst
-, libxcb, libuuid, libxshmfence, nspr, nss, pango, mesa
-
-, systemd
+{ lib, stdenv, fetchFromGitHub, fetchYarnDeps, makeDesktopItem
+, copyDesktopItems, desktopToDarwinBundle, fixup_yarn_lock, makeWrapper, nodejs
+, yarn
+, electron
```

```suggestion
{ lib
, stdenv
, fetchFromGitHub
, fetchYarnDeps
, makeDesktopItem
, copyDesktopItems
, desktopToDarwinBundle
, fixup_yarn_lock
, makeWrapper
, nodejs
, yarn
, electron
```

I suggest put each attribute in a single line here. It helps in future diffs.